projects/core/components/root/root.style.less

```diff
@@ -13,15 +13,18 @@
     }
 }
 
-html,
-.tui-zero-scrollbar {
+html[data-tui-theme] {
```

Comment:
I wonder if we should also hide global `tui-scroll-controls` if there is no `[data-tui-theme]` attribute.

Reply:
fixed

Reply:
Where? I don't see related changes.